client/src/components/Carousel.js

```diff
@@ -0,0 +1,36 @@
+import "../components/Carousel.css"
+import {FaArrowAltCircleRight, FaArrowAltCircleLeft} from "react-icons/fa";
+import {useState} from "react";
+
+export default function Carousel({images}) {
+    const [picture, setPicture] = useState(0)
+    const length = images.length
+
+    const nextSlide = () => {
+        setPicture(picture === length - 1 ? 0 : picture + 1)
```

Comment:
You could just replace this with:
```suggestion
        setPicture((picture +1) % images.length )
```
and similar but a bit different for the `previousSlide` function. Check out this article about how it works: https://dev.to/ranewallin/this-simple-math-hack-lets-you-create-an-image-carousel-without-any-if-statements-5chj